### PR TITLE
Example with a long list of amp-instagram

### PR DESCRIPTION
**Pull request by kul3r4**
#63

### CODE REVIEW

File: src/30_Advanced/Long_list_of_amp-instagram_embeds.html [hunk 1]
```diff
@@ -0,0 +1,150 @@
+<!-- 
+  #### Introduction
+
+  A page with a long list of `amp-instagram` embeds.
```
Comment (sebastianbenz) [inline src/30_Advanced/Long_list_of_amp-instagram_embeds.html]: Can you explain why this is relevant?

Comment (kul3r4) [inline src/30_Advanced/Long_list_of_amp-instagram_embeds.html]: what do you think about: "A page with a long list of `amp-instagram` embeds to show how fast and responsive amp pages are, this example contains 15 `amp-instagram` embeds."

**Comment by sebastianbenz** [comment]
Can you please fix the typo in the commit message. 

@adewale PTAL

**Comment by adewale** [comment]
It all looks good.

One request:
Can we add at least 1 image that isn't square. Here's one I made for just such an occasion: https://www.instagram.com/p/BC8yp7AxJXj/

File: src/30_Advanced/Long_list_of_amp-instagram_embeds.html [hunk 2]
```diff
@@ -0,0 +1,150 @@
+<!-- 
+  #### Introduction
+
+  A page with a long list of `amp-instagram` embeds to show how fast and responsive amp pages are, this example contains 15 `amp-instagram` embeds.
```
Comment (sebastianbenz) [inline src/30_Advanced/Long_list_of_amp-instagram_embeds.html]: s/amp/AMP/

**Comment by kul3r4** [comment]
I signed it!

2016-03-14 17:14 GMT+00:00 googlebot <notifications@github.com>:

> Thanks for your pull request. It looks like this may be your first
> contribution to a Google open source project. Before we can look at your
> pull request, you'll need to sign a Contributor License Agreement (CLA).
>
> [image: :memo:] *Please visit https://cla.developers.google.com/
> <https://cla.developers.google.com/> to sign.*
>
> Once you've signed, please reply here (e.g. I signed it!) and we'll
> verify. Thanks.
> ------------------------------
>
>    - If you've already signed a CLA, it's possible we don't have your
>    GitHub username or you're using a different email address. Check your
>    existing CLA data <https://cla.developers.google.com/clas> and verify
>    that your email is set on your git commits
>    <https://help.github.com/articles/setting-your-email-in-git/>.
>    - If you signed the CLA as a corporation, please let us know the
>    company's name.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ampproject/amp-by-example/pull/81#issuecomment-196418006>
> .
>



-- 
Chiara Chiappini

**Comment by kul3r4** [comment]
@sebastianbenz PTAL

**Comment by sebastianbenz** [comment]
LGTM

**Pull request merged**